api/v1alpha2/cloudprovider/doc.go

```diff
@@ -26,4 +26,4 @@ limitations under the License.
 // support reflecting a struct with a field of type "map[string]TYPE" to INI.
 //
 // +kubebuilder:object:generate=true
-package cloud
```

Comment:
renamed this package since golint complains about `cloud` stuffering with any other struct starting with `Cloud`, plus I think `cloudprovider` is a slighlty better name for this package anyways.